packages/manager/apps/pci-rancher/src/pages/onboarding/Onboarding.page.tsx

```diff
@@ -6,27 +6,26 @@ import {
   PageLayout,
 } from '@ovh-ux/manager-react-components';
 import { useNavigate, useParams } from 'react-router-dom';
+import { useOvhTracking } from '@ovh-ux/manager-react-shell-client';
 import onboardingImgSrc from '@/assets/onboarding-img.png';
 import Breadcrumb from '@/components/Breadcrumb/Breadcrumb.component';
 import { getCreateRancherUrl } from '@/utils/route';
-import {
-  useTrackingAction,
-  useTrackingPage,
-} from '@/hooks/useTrackingPage/useTrackingPage';
-import { TrackingEvent, TrackingPageView } from '@/utils/tracking';
+import { TrackingEvent } from '@/utils/tracking';
 import { useGuideUtils } from '@/hooks/useGuideLink/useGuideLink';
 
 export default function Onboarding() {
+  const { trackClick, trackCurrentPage } = useOvhTracking();
   const { t } = useTranslation('onboarding');
   const link = useGuideUtils();
   const navigate = useNavigate();
   const { projectId } = useParams();
   const title: string = t('title');
   const description: string = t('description');
-  useTrackingPage(TrackingPageView.Onboarding);
-  const trackAction = useTrackingAction();
+  trackCurrentPage();
```

Comment:
Question: Is tracking on pages mandatory?
I mean, this should be only handled at the route level, or am I missing something?
By doing this, I have the sensation that tracking is initialized two times.

Reply:
Yes it will send twice the tracking of the page if the Onboarding page is under Layout.component.tsx